src/js/components/ControlsSettings/ControlsSettings.tsx

```diff
@@ -140,21 +141,22 @@ const ControlsSettings = (props: any) => {
         return (
           <>
             <p className="settings-heading">{settingsObj.label}</p>
-            <a className="settings-times" onClick={settingsObj.action}><Times /></a>
+            <button className="settings-times" onClick={settingsObj.action}><Times /></button>
           </>
         )
       case 'collapsible_item':
         return (
-          <Collapsible trigger={(
-            <div className="settings-item active-setting">
-              <span className="settings-action-icon">{renderIcon(settingsObj.label)}</span>
-              <span className="settings-text">{settingsObj.label}</span>
-              <div className="flex-spacer" />
-              <span className="settings-chevron-icon">
-                <SolidArrowRight />
-              </span>
-            </div>
-          )}>
+          <Collapsible
+            trigger={(
+              <button className="settings-item active-setting">
```

Comment:
@gsin11 see if we can add `aria-expanded` too for collapsible items here?

Reply:
Yeah I was trying to add that but this plugin doesn’t have support for it, can we take this in next ticket?

Reply:
Or May be, I can remove this module and use my own?